tpm2/tpm2.go

```diff
@@ -188,49 +188,62 @@ func ReadClock(rw io.ReadWriter) (uint64, uint64, error) {
 	return decodeReadClock(resp)
 }
 
-func decodeGetCapability(in []byte) (Capability, []tpmutil.Handle, error) {
+func decodeGetCapability(in []byte) (Capability, []interface{}, bool, error) {
```

Comment:
the `Capability` return can be dropped, it's not used by the caller

Reply:
Unfortunately this is untrue. I wanted to remove it and then I ended up re-adding it because the test actually checks if the byte slice contains the right parsed capability.

Reply:
That check can be removed from the test